components-api/src/main/scala/pl/touk/nussknacker/engine/api/typed/TypeEncoders.scala

```diff
@@ -22,10 +22,13 @@ object TypeEncoders {
   private val encodeUnknown = JsonObject("refClazzName" -> fromString(classOf[Object].getName),
     "params" -> fromValues(Nil))
 
+  private val encodeNull = encodeUnknown
```

Comment:
I wonder if it's ok. Will show up in FE as Unknown? Maybe we'd better write sth specific?

Reply:
Deserialization and showing this on FE will work fine thanks to `typeField` and `display` fields added in `encodeTypingResult`. I am not sure what these fields are used for, but Object seems like good choice since Null can represent any type. I will add a comment.